app/channels/user_channel.rb

```diff
@@ -1,6 +1,6 @@
 class UserChannel < ApplicationCable::Channel
   def subscribed
-    if current_user != nil
+    if !current_user.nil?
```

Comment:
This doesn't read that well. What about using an `unless` or better yet just `if current_user`